### DISCUSSION TITLE
backports.zoneinfo._common.ZoneInfoNotFoundError: 'No time zone found with key Europe/Kyiv'

### DISCUSSION THREAD

**Discussion by julien-flume** [netbox-community/netbox#10022] [General]
Hi guys,

I'm trying to update my netbox from 2.10.4 to 3.2.8 and I go this error during the upgrade 👍 

backports.zoneinfo._common.ZoneInfoNotFoundError: 'No time zone found with key Europe/Kyiv'

Do you know how I can solve this problem and finish my upgrade ? Or maybe, do I have to follow an upgrade path ?

Regards,

Julien

**Comment by jeremystretch** [comment]
Please see #9986

**Reply by julien-flume** [reply]
Thx Jeremy

**Reply by candlerb** [reply]
Also remember that it is mandatory to upgrade to 2.11.x first, *then* to 3.x
